domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1076,15 +1137,15 @@ public void saveBatchJsonFail(TestContext context) {
 
   @Test
   public void saveBatchJsonNullArray(TestContext context) {
-    createFoo(context).saveBatch(BAR, (JsonArray)null, context.asyncAssertSuccess(save -> {
-      context.assertEquals(0, save.getNumRows());
+    createFoo(context).saveBatch(FOO, (JsonArray)null, context.asyncAssertSuccess(save -> {
+      context.assertNull(save);
```

Comment:
This is an unneccessary breaking change that is inconsistent with vertx-pg-client that returns an empty RowSet when "UPDATE ... WHERE ..." does not update any record.
Can you return an empty RowSet?

Reply:
ok